### ISSUE TITLE
Allow changing players' (permisssion) groups (through PEX API or something) based on their status in game (alive, dead, spectator)

### ISSUE THREAD

**Issue by Amunak** [AmauryCarrade/UHPlugin#83]
This would be an awesome feature. Whille you can limit users' commands through SpectatorPlus you can't give them a prefix or give them *more* commands. It would be great if you could assign a group to a player that is spectating from the beginning or who just died (you could then even stop using SpectatorPlus and could just give players teleport or something).

**Comment by Amauryroco** [comment]
I don't understand what you mean xD
But that's not my job to understand, so I just hope it's an awesome feature you want @AmauryCarrade to add. :)

**Comment by Amunak** [comment]
Yeah sorry I wrote it in a hurry. Basically what I want is when the PermissionEx plugin (or some other? Maybe there is a universal solution to this) is present then when a player dies it gets assigned a configurable PEX permission group. Same for when the game starts and they are a player. It would mean that a player that is alive and a player that is dead could have entirely different permissions.

Oh and it would also be lovely if you could run some commands on the players when they die and stuff (be it just stuff like /say {deadPlayer} is a noob! or something more complicated like /vanish on {deadPlayer}).

**Comment by Amauryroco** [comment]
@Amunak Oh okay, I see what you mean ! It's interesting but I don't know if it's going to be useful...
